packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -521,10 +519,6 @@ export default class AnalysisWebWorker {
 			}
 		}
 
-		if ( useKeywordDistribution && isUndefined( assessor.getAssessment( "keyphraseDistribution" ) ) ) {
-			assessor.addAssessment( "keyphraseDistribution", keyphraseDistribution );
-		}
-
```

Comment:
There are also still some references to `useKeywordDistribution` in this file. They also need to be removed since they're obsolete :)